Functional.Tests/Utilities/Assertions/OptionTypeAssertions.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using FluentAssertions;
+using FluentAssertions.Equivalency;
+using FluentAssertions.Execution;
+
+namespace Functional.Tests.Utilities.Assertions
+{
+	/// <summary>
+	/// Defines assertions for <see cref="Option{TValue}"/> type.
+	/// </summary>
+	/// <typeparam name="T">The contained type.</typeparam>
+	public class OptionTypeAssertions<T>
```

Comment:
I'm unsure of whether these provide any advantage over AssertSome() and AssertNone().

Reply:
As far as I can tell, those methods do not exist.  Only `AssertSuccess` and `AssertFailure` were defined, but (as I mentioned elsewhere) I deleted those in favour of the FluentAssertions syntax.